crates/nargo/tests/test_data/2_div/src/main.nr

```diff
@@ -0,0 +1,16 @@
+mod foo;
```

Comment:
Lets delete the multiline comment from this file. In the future, we should include comments which say what it actually happening and what we are testing. Can you do this for all of the examples?

Reply:
yes, it's done